src/Parser/Priest/Shadow/Modules/Spells/VoidformAverageStacks.js

```diff
@@ -48,15 +49,34 @@ class VoidformAverageStacks extends Analyzer {
 
     const lastVoidformWasExcluded = voidforms[voidforms.length - 1].excluded;
 
-    return (<StatisticBox
+    return (<ExpandableStatisticBox
```

Comment:
I recommend an enter after the parenthesis, this keeps indentation sane and easier to follow.